cita-executor/src/main.rs

```diff
@@ -157,14 +157,20 @@ impl Options {
 }
 
 fn main() {
-    micro_service_init!("cita-executor", "CITA:executor");
     let matches = App::new("executor")
         .version(get_build_info_str(true))
         .long_version(get_build_info_str(false))
         .author("Cryptape")
         .about("CITA Block Chain Node powered by Rust")
-        .arg_from_usage("-c, --config=[FILE] 'Sets a switch config file'")
+        .args_from_usage(
+            "-c, --config=[FILE] 'Sets a switch config file'
+                          -s, --stdout 'Log to console'",
+        )
         .get_matches();
+
+    let stdout = matches.is_present("stdout");
+    micro_service_init!("cita-executor", "CITA:executor", stdout);
+
```

Comment:
why executor missing this log info?
`info!("Version: {}", get_build_info_str(true));`

Reply:
I know. In the code block next.

Reply:
My fault， I add it.

Reply:
it existed at 176 line.